config/src/config/consensus_config.rs

```diff
@@ -110,10 +110,10 @@ impl ConsensusConfig {
 
     pub fn load(&mut self) -> Result<()> {
         if !self.consensus_keypair_file.as_os_str().is_empty() {
-            self.consensus_keypair = ConsensusKeyPair::load_config(self.consensus_keypair_file());
+            self.consensus_keypair = ConsensusKeyPair::load_config(self.consensus_keypair_file())?;
```

Comment:
I'm starting to think that the best thing to do would be to have keypair types be Option<T> so that its clear when we haven't loaded the keys yet.

Reply:
This is a good point. I'll add it to my list of Config issues.